### PR TITLE
Make icons esm enabled

### PR DESCRIPTION
**Pull request by selbekk**
This pull request makes the ffe-icons-react package code-splittable, by providing an esm target as well as a CJS one.

### CODE REVIEW

File: packages/ffe-icons-react/.babelrc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+                ["env", {
+                    "modules": false,
+                    "targets": { "browsers": "ie 11, last 2 versions" }
+                }],
+                "stage-3",
+                "react"
+            ]
+        }
+    }
+}
```
Comment (selbekk) [inline packages/ffe-icons-react/.babelrc:23]: This can be simplified greatly by upgrading to Babel 7. I'll attack that task tomorrow.

Comment (henrikhermansen) [reply packages/ffe-icons-react/.babelrc:23]: Yes! Do it! :)
Beware you might need a dependency `"babel-core": "7.0.0-bridge.0"`, unless jest has released a new version that doesn't depend on babel 6.

**Review by ivarni — APPROVED** [review]
noise 1

**Comment by selbekk** [comment]
This could (should?) probably be done for all of our components, in order to improve the final bundle size even more.

**Pull request merged**